docs/documentation/stories.md

```diff
@@ -20,3 +20,4 @@
  - [Routing](stories/routing)
  - [3rd Party Lib](stories/third-party-lib)
  - [Corporate Proxy](stories/using-corporate-proxy)
+ - [Serve from Disk](stories/disk-serve)
```

Comment:
Nit, but you say "serve from disk" here and later it is "serving from disk" I'd pick and be consistent.